pulp_ostree/app/viewsets.py

```diff
@@ -102,12 +102,29 @@ class OstreeDistributionViewSet(core.DistributionViewSet):
     serializer_class = serializers.OstreeDistributionSerializer
 
 
+class RefFilter(BaseFilterSet):
+    """A filterset class for refs."""
+
+    class Meta:
+        model = models.OstreeRef
+        fields = {"name": ["exact"]}
```

Comment:
I would use the NAME_FILTER_OPTIONS constant.

https://github.com/pulp/pulpcore/blob/de67d41c39d9d4bde2e8f86e4bf3f51d6cc774e7/pulpcore/app/viewsets/base.py#L24

Reply:
Hmm, will do, thanks.